core/domain/subtopic_page_domain.py

```diff
@@ -89,45 +103,54 @@ def __init__(
         self.recorded_voiceovers = recorded_voiceovers
         self.written_translations = written_translations
 
-    def validate(self):
+    def validate(self) -> None:
         """Validates the SubtopicPageContentsObject, verifying that all
         fields are of the correct type.
         """
-        self.subtitled_html.validate()
+        self.subtitled_html.validate( # type: ignore[no-untyped-call]
+        )
         content_ids = set([self.subtitled_html.content_id])
-        self.recorded_voiceovers.validate(content_ids)
-        self.written_translations.validate(content_ids)
+        self.recorded_voiceovers.validate( # type: ignore[no-untyped-call]
+        content_ids)
+        self.written_translations.validate( # type: ignore[no-untyped-call]
+        content_ids)
 
     @classmethod
-    def create_default_subtopic_page_contents(cls):
+    def create_default_subtopic_page_contents(cls) -> 'SubtopicPageContents':
         """Creates a default subtopic page contents object.
 
         Returns:
             SubtopicPageContents. A default object.
         """
         content_id = feconf.DEFAULT_SUBTOPIC_PAGE_CONTENT_ID
         return cls(
-            state_domain.SubtitledHtml.create_default_subtitled_html(
+            state_domain.SubtitledHtml.create_default_subtitled_html( # type: ignore[no-untyped-call]
                 content_id),
-            state_domain.RecordedVoiceovers.from_dict(
+            state_domain.RecordedVoiceovers.from_dict( # type: ignore[no-untyped-call]
                 {'voiceovers_mapping': {content_id: {}}}),
-            state_domain.WrittenTranslations.from_dict(
+            state_domain.WrittenTranslations.from_dict( # type: ignore[no-untyped-call]
                 {'translations_mapping': {content_id: {}}}))
 
-    def to_dict(self):
+    def to_dict(self) -> SubtopicPageContentsDict:
         """Returns a dict representing this SubtopicPageContents domain object.
 
         Returns:
             dict. A dict, mapping all fields of SubtopicPageContents instance.
         """
         return {
-            'subtitled_html': self.subtitled_html.to_dict(),
-            'recorded_voiceovers': self.recorded_voiceovers.to_dict(),
-            'written_translations': self.written_translations.to_dict()
+            'subtitled_html': self.subtitled_html.to_dict( # type: ignore[no-untyped-call]
+            ),
+            'recorded_voiceovers': self.recorded_voiceovers.to_dict( # type: ignore[no-untyped-call]
+            ),
+            'written_translations': self.written_translations.to_dict( # type: ignore[no-untyped-call]
+            )
         }
 
     @classmethod
-    def from_dict(cls, page_contents_dict):
+    def from_dict(
+        cls,
+        page_contents_dict: Dict[str, object]
+        ) -> 'SubtopicPageContents':
```

Comment:
```suggestion
        cls,
        page_contents_dict: Dict[str, object]
    ) -> SubtopicPageContents:
```

will this work? If not just add the quotes but keep the indentation.